pkg/AnewLibrary/Design/Atoms/ConnectionInspector.js

```diff
@@ -40,17 +41,22 @@ stratifyTypes(prefixId, propName, type, candidates, connection) {
   return choices;
 },
 getTypeMatch(propA, propB) {
+  let match = 0;
+  const intersect = (a, b) => {
+    var setB = new Set(b);
+    return [...new Set(a)].filter(x => setB.has(x));
+  };
   const objectTypes = ['Pojo', 'Json'];
   const basicTypes = ['String', 'Text', 'Number'];
   const booleanTypes = ['Nonce', 'Boolean'];
-  let match = 0;
-  if (propA.type === propB.type) {
+  const [A, B] = [propA.type, propB.type];
+  if (A === B || (intersect(A.split('|'), B.split('|')))) {
```

Comment:
the delimiter used to be `:`
i'm fine either way (in fact i confused them multiple times)... 
I'll fix the rest of atom-manifests that use `:` in a followup pr

Reply:
Ty, I didn't really mean to change it, but my brain kept saying "it's this OR that", so I would type "|".